src/main/java/io/jenkins/plugins/analysis/warnings/PhpStan.java

```diff
@@ -0,0 +1,63 @@
+package io.jenkins.plugins.analysis.warnings;
+
+import edu.hm.hafner.analysis.parser.checkstyle.CheckStyleParser;
+import edu.umd.cs.findbugs.annotations.NonNull;
+
+import org.kohsuke.stapler.DataBoundConstructor;
+import org.jenkinsci.Symbol;
+import hudson.Extension;
+
+import io.jenkins.plugins.analysis.core.model.ReportScanningTool;
+
+/**
+ * Provides a parser and customized messages for PHPStan. Delegates to {@link CheckStyleParser}.
+ *
+ * @author Jeroen Jans
+ */
+public class PhpStan extends ReportScanningTool {
+    private static final long serialVersionUID = 2699509705079011738L;
+
+    static final String ID = "phpstan";
```

Comment:
private

Reply:
(I changed that in all other parsers as well:-)

Reply:
I've updated the visibility to private.
I've also rebased on master to resolve the merge conflict in the SUPPORTED-FORMATS.md file